src/mcts/search.cc

```diff
@@ -384,11 +385,14 @@ std::vector<std::string> Search::GetVerboseStats(Node* node) const {
   for (const auto& edge : node->Edges()) edges.push_back(edge);
 
   std::sort(edges.begin(), edges.end(),
-            [&fpu, &U_coeff, &draw_score](EdgeAndNode a, EdgeAndNode b) {
+            [&fpu, &U_coeff, &draw_score, &lcb_percentile](EdgeAndNode a,
```

Comment:
A couple of overfull lines, but no idea what the correct indentation is supposed to be.